src/internal/storage/fileset/postgres_cache.go

```diff
@@ -0,0 +1,122 @@
+package fileset
+
+import (
+	"context"
+
+	proto "github.com/gogo/protobuf/proto"
+	"github.com/gogo/protobuf/types"
+	"github.com/jmoiron/sqlx"
+	"github.com/pachyderm/pachyderm/v2/src/internal/dbutil"
+	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
+	"github.com/pachyderm/pachyderm/v2/src/internal/pachsql"
+	"github.com/pachyderm/pachyderm/v2/src/internal/storage/track"
+)
+
+// CreatePostgresCacheV1 creates the table for a cache.
+// DO NOT MODIFY THIS FUNCTION
+// IT HAS BEEN USED IN A RELEASED MIGRATION
+func CreatePostgresCacheV1(ctx context.Context, tx *pachsql.Tx) error {
+	const schema = `
+	CREATE TABLE storage.cache (
+		key text NOT NULL PRIMARY KEY,
```

Comment:
This could also be a `BYTEA`.  Are you hex-encoding the IDs?

Reply:
The task IDs are hex encoded. What would be the benefit of `BYTEA` if the key field is expected to be small and printable characters? I'm just wondering if the performance / space improvements is going to outweigh the additional complexity that comes with managing binary data.

Reply:
I thought the task IDs were just `pachhash.Output`s?

RE performance: Just 2x the storage and 2x over the wire.  I was a little concerned about that for the chunks and filesets, but this table will be smaller. 

I'm familiar with the human readability argument.  It makes the most sense when you are debugging in a SQL shell.  Encoding can sometimes lead to issues double encoding/unencoding.  Although that's more common in languages other than Go where `[]byte` is clearly used for binary and all the hex strings are `string`s.

Reply:
No, they are hex encoded. We use pachhash, then hex encode immediately and use that string everywhere else. It seems like the benefit of using strings everywhere outweighs the cost of 2x data for keys this small.

Reply:
I think using strings everywhere is actually a drawback in terms of lost typing information.  The `String` method allows the pachhash.Output to print nicely.  And you can always convert to a string at the last minute before you do anything.

With regards to the column type, TEXT is fine.